arwen/host/arwen.go

```diff
@@ -382,6 +382,11 @@ func (host *vmHost) RunSmartContractCall(input *vmcommon.ContractCallInput) (vmO
 	return
 }
 
+// Close closes all internal instances of the vm
+func (host *vmHost) Close() error {
```

Comment:
comment does not match the behaviour

Reply:
it does not need to close anything, but it the same as for new vm. this is a feat only for compatibility